test/Base_Tests/src/Semantic/Meta_Spec.enso

```diff
@@ -209,6 +224,24 @@ add_specs suite_builder =
             e_tpe . should_equal_type IOException
             e_tpe . should_not_equal_type JException
 
+        group_builder.specify "multi_value_as_text" <|
+            v = R.V "hi" 3 . as_text
+            Meta.type_of v . should_equal Text
+
+        group_builder.specify "multi_value_as_int" <|
+            v = R.V "hi" 3 . as_int
+            Meta.type_of v . should_equal Integer
+
+        group_builder.specify "multi_value_as_text_int" <|
+            v = R.V "hi" 3 . as_text_int
+            Meta.type_of v . should_equal Text
```

Comment:
Shouldn't you also add test for `Meta.type_of v . should_equal Integer` ?

Reply:
I cannot right now, as the type doesn't equal `Integer`. Currently only the first type is used by `type_of`. That's likely to change with the work on 
- #11482

For now, the test checks just the first type.